### DISCUSSION TITLE
Prevent health alarm during a specific time of day (daily maintenance window)

### DISCUSSION THREAD

**Discussion by danmichaelo** [netdata/netdata#13414] [Q&A] [answered]
Hi!

We have a server that runs some heavy maintenance tasks every day that cause some alarms to go off. Is there a way to prevent alarms to go off in a certain time window? Something like this pseudo-code:

    crit: $this > 0.5 && !($timeOfDay > "01:00" && $timeOfDay < "01:15")

**Comment by MrZammler** [comment, marked as answer]
Hi @danmichaelo !

While not exactly the same, there is a way to instruct Netdata to silence alerts using an API call. The help page [here](https://learn.netdata.cloud/docs/agent/web/api/health/#health-management-api) has some more information on how to achieve this.

So, you could for example incorporate in your tasks a call to Netdata to silence the alerts before the tasks begin, and restore them after they're done. Would that work in your case?

**Reply by danmichaelo** [reply]
Thanks, that's an interesting option to explore! Just have to make sure that the alarms are re-enabled if the import script crash for some reason, but I guess that can be done in one way or another!
